### PR TITLE
New Adapter: Adelement 

### PR DESCRIPTION
**Pull request by radadiapg**
adding adelement bidder adapter

### CODE REVIEW

File: src/main/java/org/prebid/server/bidder/adelement/AdelementBidder.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+            return Collections.emptyList();
+        }
+
+        return bidResponse.getSeatbid().stream()
+                .filter(Objects::nonNull)
+                .map(SeatBid::getBid)
+                .filter(Objects::nonNull)
+                .flatMap(Collection::stream)
+                .filter(Objects::nonNull)
+                .map(bid -> BidderBid.of(bid, BidType.banner, bidResponse.getCur()))
```
Comment (SerhiiNahornyi) [inline src/main/java/org/prebid/server/bidder/adelement/AdelementBidder.java:67]:
```yaml
app-media-types:
        - banner
        - video
        - audio
      site-media-types:
        - banner
        - video
        - audio
```
So, only BidType.banner or other types is possible too?

Comment (SerhiiNahornyi) [reply src/main/java/org/prebid/server/bidder/adelement/AdelementBidder.java]: Generic, also supports native type, if it's ok for you, you can use it

Comment (radadiapg) [reply src/main/java/org/prebid/server/bidder/adelement/AdelementBidder.java]: does failed Functional tests related to our code ?

**Review by SerhiiNahornyi — APPROVED** [review]
LGTM) 
Will be part of next release

**Pull request merged**